src/quat.js

```diff
@@ -447,36 +447,50 @@ export function fromMat3(out, m) {
 }
 
 /**
- * Creates a quaternion from the given euler angle x, y, z.
+ * Creates a quaternion from the given euler angle z, y, x.
  *
  * @param {quat} out the receiving quaternion
- * @param {x} Angle to rotate around X axis in degrees.
- * @param {y} Angle to rotate around Y axis in degrees.
  * @param {z} Angle to rotate around Z axis in degrees.
+ * @param {y} Angle to rotate around Y axis in degrees.
+ * @param {x} Angle to rotate around X axis in degrees.
  * @returns {quat} out
  * @function
  */
-export function fromEuler(out, x, y, z) {
+export function fromEuler(out, z, y, x) {
```

Comment:
I don't think I agree with changing the argument order -- I see you've changed the contents of the function so the result is the same but I feel changing the argument order to something other than x y and z is confusing. Euler angles are often stored in XYZ order independent of application order in my experience. I think all that should change is the documentation.

Reply:
I see now that I misinterpreted the original issue. I spent some more time reading up on Euler angles (it's been a couple years since my computer graphics class 😅 ).

> I think all that should change is the documentation.

Done, though I did refactor the internals of both into a private function to avoid duplicated code.